### PR TITLE
HIP 131: Updates

### PR DESCRIPTION
**Pull request by waveform06**
Updated with text clarifications

### CODE REVIEW

File: 0131-bridging-gap-between-verification-mappers-and-anti-gaming-measures.md [hunk 1]
```diff
@@ -61,11 +62,11 @@ None.
 
 ## Implementation
 
-Nova and Helium mobile have agreed to the making the implementation of this proposal.
+Nova Labs and Helium Mobile have agreed to the making the implementation of this proposal.
```
Comment (abhay) [inline 0131-bridging-gap-between-verification-mappers-and-anti-gaming-measures.md:65]:
```suggestion
Nova Labs and Helium Mobile have agreed to the implementation of this proposal but have not indicated a timeline. 
```

File: 0131-bridging-gap-between-verification-mappers-and-anti-gaming-measures.md [hunk 2]
```diff
@@ -11,47 +11,48 @@
...
+- SPs can reduce the Oracle Hex Boosting multiplier of a Hotspot or radios if they have reasonable evidence the POI hexes are being taken advantage of by a malicious actor (such as an inaccurate CDR).
+- Deployers can open a support ticket with the SP if they feel they have been mistakenly identified as malicious. This may involve, but not be limited to, the SP attempting a physical verification of coverage at a given location, which would involve a connection to a radio (Wi-Fi or CBRS) to verify its existence. 
+- This HIP expires after 150 epochs (approximately 5 months) starting from the community HIP implementation notification. This gives the network time to implement a programmatic solution such as HIP-118.  However, the community can choose to extend this timeframe with an additional HIP-131 extension vote if it feels sufficient progress has not been made on programmatic solutions.  It's suggested the community review the need for a vote several weeks ahead of expiration to provide time for deliberation and the general voting process.  Note that implementation of a programmatic solution would not reduce the 150 epoch duration.
```
Comment (abhay) [inline 0131-bridging-gap-between-verification-mappers-and-anti-gaming-measures.md:51]: @madninja do you think this is long enough for verification mapping implementation?

**Review by mrfizzy99 — APPROVED** [review]
Approved

**Comment by mrfizzy99** [comment]
@abhay @waveform06  I have one last change, not sure how I can edit the current revision of.  Needed to add: <insert  line 44>

- CBRS Radios are given an extra 21 epochs after the first initial advance notice of HIP-131 from Helium Mobile and Nova, to allow more time for deployers and potential mappers to install the CBRS Beta SIM in order to create a CDR for CBRS radios.

**Comment by abhay** [comment]
Need a new PR.

**Pull request merged**